lib/ansible/modules/utilities/logic/assert.py

```diff
@@ -53,6 +56,6 @@
     that:
       - "my_param <= 100"
       - "my_param >= 0"
-    msg: "'my_param' must be between 0 and 100"
+    fail_msg: "'my_param' must be between 0 and 100"
```

Comment:
also make notes in the examples that 2.7 is the version in which it starts working